scout/adapter/mongo/institute.py

```diff
@@ -96,6 +96,7 @@ def update_institute(
             "frequency_cutoff": frequency_cutoff,
             "gene_panels": gene_panels,
             "gene_panels_matching": gene_panels_matching,
+            "safe_genes_matching": self.safe_genes_filter(internal_id),
```

Comment:
@dnil gets saved in the institute:

<img width="528" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/ba953a5c-75f4-4059-82d8-5f7134236a18">